pkg/parse/json/json_test.go

```diff
@@ -0,0 +1,552 @@
+	  "compare_url": "https://api.github.com/repos/Codertocat/Hello-World/compare/{base}...{head}",
+	  "merges_url": "https://api.github.com/repos/Codertocat/Hello-World/merges",
+	  "archive_url": "https://api.github.com/repos/Codertocat/Hello-World/{archive_format}{/ref}",
+	  "downloads_url": "https://api.github.com/repos/Codertocat/Hello-World/downloads",
+	  "issues_url": "https://api.github.com/repos/Codertocat/Hello-World/issues{/number}",
+	  "pulls_url": "https://api.github.com/repos/Codertocat/Hello-World/pulls{/number}",
+	  "milestones_url": "https://api.github.com/repos/Codertocat/Hello-World/milestones{/number}",
+	  "notifications_url": "https://api.github.com/repos/Codertocat/Hello-World/notifications{?since,all,participating}",
+	  "labels_url": "https://api.github.com/repos/Codertocat/Hello-World/labels{/name}",
+	  "releases_url": "https://api.github.com/repos/Codertocat/Hello-World/releases{/id}",
+	  "deployments_url": "https://api.github.com/repos/Codertocat/Hello-World/deployments",
+	  "created_at": "2019-05-15T15:19:25Z",
+	  "updated_at": "2019-05-15T15:19:27Z",
+	  "pushed_at": "2019-05-15T15:20:32Z",
+	  "git_url": "git://github.com/Codertocat/Hello-World.git",
+	  "ssh_url": "git@github.com:Codertocat/Hello-World.git",
+	  "clone_url": "https://github.com/Codertocat/Hello-World.git",
+	  "svn_url": "https://github.com/Codertocat/Hello-World",
+	  "homepage": null,
+	  "size": 0,
+	  "stargazers_count": 0,
+	  "watchers_count": 0,
+	  "language": null,
+	  "has_issues": true,
+	  "has_projects": true,
+	  "has_downloads": true,
+	  "has_wiki": true,
+	  "has_pages": true,
+	  "forks_count": 0,
+	  "mirror_url": null,
+	  "archived": false,
+	  "disabled": false,
+	  "open_issues_count": 2,
+	  "license": null,
+	  "forks": 0,
+	  "open_issues": 2,
+	  "watchers": 0,
+	  "default_branch": "master"
+	},
+	"sender": {
+	  "login": "Codertocat",
+	  "id": 21031067,
+	  "node_id": "MDQ6VXNlcjIxMDMxMDY3",
+	  "avatar_url": "https://avatars1.githubusercontent.com/u/21031067?v=4",
+	  "gravatar_id": "",
+	  "url": "https://api.github.com/users/Codertocat",
+	  "html_url": "https://github.com/Codertocat",
+	  "followers_url": "https://api.github.com/users/Codertocat/followers",
+	  "following_url": "https://api.github.com/users/Codertocat/following{/other_user}",
+	  "gists_url": "https://api.github.com/users/Codertocat/gists{/gist_id}",
+	  "starred_url": "https://api.github.com/users/Codertocat/starred{/owner}{/repo}",
+	  "subscriptions_url": "https://api.github.com/users/Codertocat/subscriptions",
+	  "organizations_url": "https://api.github.com/users/Codertocat/orgs",
+	  "repos_url": "https://api.github.com/users/Codertocat/repos",
+	  "events_url": "https://api.github.com/users/Codertocat/events{/privacy}",
+	  "received_events_url": "https://api.github.com/users/Codertocat/received_events",
+	  "type": "User",
+	  "site_admin": false
+	}
+}`,
```

Comment:
this is pretty minor but it might be easier to read this if we put the payload into separate files?